projects/fsastorefrontlib/src/assets/translations/en/quote.en.ts

```diff
@@ -29,5 +29,75 @@ export const quote = {
       ERROR: 'Error',
       na: 'N/A',
     },
+    quoteSummary: 'Quote Summary',
+    details: {
+      insurances_travel: {
+        trip: 'Travel Information',
+        tripDestination: 'Destination',
+        Travellers: 'Number of Travelers',
+        tripStartDate: 'Start Date',
+        traveller: 'Traveler\'s Name', // prettier-ignore
```

Comment:
If this comment is for prettier to ignore warning you can maybe use template literals ([https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals]) and remove that comment:

traveller: `Traveler\'s Name`

Reply:
Done!